app-arch/bzip2/bzip2-1.0.6.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic ltprune multilib-minimal
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${P}.tar.gz"
```

Comment:
I don't know this upstream source. Unless I know and understand the reason why we should switch we will stick for official upstream.

Reply:
well, this upstream is the same as far as actual code goes. It simply has the needed stuff for autotools added in, and builds the same binaries. If you don't trust it, you can easily dl both and do hash comparisons of the contents of the tarball.